netlogo-core/src/main/api/LabProtocol.scala

```diff
@@ -14,7 +14,7 @@ case class LabProtocol(name: String,
                     metrics: List[String],
                     valueSets: List[RefValueSet])
 {
-  def countRuns = repetitions * valueSets.map(_.toList.size).product
+  val countRuns = repetitions * valueSets.foldLeft(1)((accum,elt) => accum * elt.length.toInt)
```

Comment:
this got back in? nbd I guess

Reply:
Hi @SethTisue. That's my bad since this is leftover from fixing the SteppedValueSets issue with space. This should have been:
```
.map(_.length.toInt).product
```